mods/action.js

```diff
@@ -155,6 +155,9 @@ ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
     body.action = defBody.action;
     body.format = body.format || defBody.format || 'json';
     body.formatversion = body.formatversion || defBody.formatversion || 1;
+    if (defBody.rawcontinue && !body.continue) {
```

Comment:
the `continue` param must be an empty string on first invocation, so this will evaluate to true always. Use:

```js
if (!body.hasOwnProperty('continue') && defBody.rawcontinue)
```